dpsim/src/KLUAdapter.cpp

```diff
@@ -18,6 +18,7 @@ KLUAdapter::~KLUAdapter()
         klu_free_symbolic(&mSymbolic, &mCommon);
     if (mNumeric)
         klu_free_numeric(&mNumeric, &mCommon);
+    mSLog->info("Number of Pivot Faults: {}", mPivotFaults);
```

Comment:
This line might need to use the macro like `SPDLOG_LOGGER_INFO(mSLog, ...);`

Reply:
Indeed, thanks!